### DISCUSSION TITLE
360 image in a scene getting inverted in browser, but only some times

### DISCUSSION THREAD

**Discussion by Charlesc22** [mozilla/hubs#2853] [General Questions & Discussion] [answered]
Hey guys, I made a scene with a 360 pano and it works fine, but sometimes when a user loads the room in his browser the pano is upside down, what should I do to avoid this from happening? What determines the orientation of the image?

Thanks

**Comment by Charlesc22** [comment]
Ok, I tested more and it reproducible. Very simple actually
1- In FireFox (PC), the pano image displays correctly
2- In Chfrome (PC), the pano is upside down
3- In Edge(PC), the pano is also upside down

**Comment by emclaren** [comment, marked as answer]
Hi! this seems more like a bug. I've created an issue for this here: https://github.com/mozilla/hubs/issues/2925
